src/js/models/state.model.js

```diff
@@ -1,43 +1,79 @@
 export class StateModel {
-    state;
+    refreshState = {
+        restart: false,
+        reset: false,
+    };
```

Comment:
(마이너) stateModel이니 필드명에 state가 꼭 포함될필요는 없을 것 같아요
```suggestion
    condition = {
      restarted: false,
      reseted: false,
    }
```